sympy/stats/rv.py

```diff
@@ -1083,50 +1083,79 @@ def sample(expr, condition=None, size=(), library='scipy',
     >>> X, Y, Z = Die('X', 6), Die('Y', 6), Die('Z', 6) # Finite Random Variable
 
     >>> die_roll = sample(X + Y + Z) # doctest: +SKIP
-    >>> next(die_roll) # doctest: +SKIP
+    >>> die_roll # doctest: +SKIP
```

Comment:
Can the skip be removed now?

Reply:
```python
File "/home/aman/sympy/sympy/stats/rv.py", line 1086, in sympy.stats.rv.sample
Failed example:
    die_roll
Expected:
    6
Got:
    7
```
I dont think so that they can be removed since the output is random.

Reply:
Maybe set the seed at the start of the docstring?

Reply:
Please guide me with that. Do you mean specifying a seed in every example like `die_roll = sample(X+Y+Z, seed = 12)`. That would indeed output a consistent result if we specify seed in every example.

Reply:
Is there not a way to set the seed just once and have it used in subsequent calls to sample?

Reply:
I guess this is why it would be good to have a `sampler` class: it could store the seed and then you could do:
```python
state = sampler('numpy', seed)
state.sample(X + Y + Z)
state.sample(X)
```
For now I guess let's just leave the `+SKIP` there.

Reply:
Apart from this docstring, where else a change in documentation is required?